### DISCUSSION TITLE
[5주차] 온라인 스터디

### DISCUSSION THREAD

**Discussion by ChaerinYu** [no-study-no-future/interview-study#41] [General]
### 진행순서
~ 20:15 각자 PR리뷰
~ 21:00 발표
~ 21:10 회고 (좋았던 점, 건의사항, 느낀점 등)

**Comment by ChaerinYu** [comment]
### 발표
1. 유채린
 * [Database] 정규화 Normalization #38 
2. 이주형
 * [Database] Isolation Level(격리 수준, 고립 수준) #35 
3. 김정윤
 * [Java/Algorithm] 다익스트라(Dijkstra) 알고리즘 #40 
4. 이동호
 * [Java] Java Reflection #36

**Comment by ChaerinYu** [comment]
### 회고 (a.k.a. 출석체크)
* 참석 확인용이므로 `감사합니다!` 만 작성해주셔도 좋아요 😊
* 좋았던 점
* 건의사항
* 느낀점

**Reply by doho-ho** [reply]
출석이용 
- 정신없다 현대사회 @_@
- 돌아와요 유채린

**Reply by leeejuhyeong** [reply]
- 동시성 제어에 대해 준비하면서 미흡했던 level 3 repetable read를 보충하도록 하겠습니다..

**Reply by KJY97** [reply]
- 시간복잡도 추가하겠습니다!
- 앞으로 우리의 스터디는 어떻게 되는 것인가??

**Reply by ChaerinYu** [reply]
* 알고리즘은 항상 까먹네요..^-^
* 오늘도 새로운 지식 많이 얻어가요~~
